source/tyr/tyr/helper.py

```diff
@@ -43,9 +43,11 @@
 from flask import json
 from jsonschema import validate, ValidationError
 from tyr.formats import instance_config_format
+import requests
 
 END_POINT_NOT_EXIST_MSG = 'end_point doesn\'t exist'
 BILLING_PLAN_NOT_EXIST_MSG = 'billing plan doesn\'t exist'
+MAX_NB_CALLS = 3
```

Comment:
Maybe a more specific name ?
```suggestion
MAX_TRY_FOR_REPOST_TO_SECONDARY_TYR= 3
```

Reply:
Same, I'd push to:
```suggestion
MAX_TRY_FOR_REPOST_TO_URL = 3
```
(maybe `EXTERNAL_URL`,  `ANOTHER_URL`, `SECONDARY_URL` if `URL` is not clear enough)